js/augurs-prophet-js/src/lib.rs

```diff
@@ -162,6 +162,48 @@ impl Prophet {
         let predictions = self.inner.predict(data)?;
         Ok(Predictions::from((self.level, predictions)))
     }
+
+    /// Add a custom seasonality to the model
+    #[wasm_bindgen(js_name = "addSeasonality")]
+    pub fn add_seasonality(
+        &mut self,
+        name: String,
+        seasonality: Seasonality,
+    ) -> Result<(), JsError> {
+        self.inner.add_seasonality(name, seasonality.into())?;
+        Ok(())
+    }
+}
+
+/// Can be used to specify custom seasonality
+#[derive(Debug, Clone, Deserialize, Serialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+#[tsify(from_wasm_abi, into_wasm_abi, type_prefix = "Prophet")]
+pub struct Seasonality {
+    period: PositiveFloat,
```

Comment:
This will need a `#[tsify(type = "number")]` attribute on it otherwise the generated Typescript types refer to a non-existent (in JS) `PositiveFloat` type.